packages/manager/apps/pci-volume-snapshot/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "@ovh-ux/manager-pci-volume-snapshot-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "PCI Volume Snapshot react app",
+  "type": "module",
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json && vite build",
+    "coverage": "vitest run --coverage",
+    "dev": "vite",
+    "lint": "eslint ./src",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-pci-volume-snapshot-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-pci-volume-snapshot-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-pci-volume-snapshot-app' --include-dependencies -- npm run dev:watch --if-present",
+    "test": "vitest run"
```

We can consider increasing performance on execution (https://vitest.dev/guide/improving-performance.html#pool)
```suggestion
    "test": "vitest run --pool=threads"
```

---

Done :+1: